### DISCUSSION TITLE
how to turn conversation turn limits off?

### DISCUSSION THREAD

**Discussion by legaltextai** [open-webui/open-webui#4608] [Q&A]
I am limited by 4 for all models. Which goes away when I disable "Vision" for the model.  
Thank you

**Comment by legaltextai** [comment]
do I need to install a this pipeline to overrun the defaults? 
https://github.com/open-webui/pipelines/blob/main/examples/filters/conversation_turn_limit_filter.py

**Comment by justinh-rahb** [comment]
There is no built-in turn limit. Screenshots, and log files are usually more helpful for debugging.

**Comment by legaltextai** [comment]
Ended up installing that pipeline and set the limits. Cheers,

**Comment by Aleksahek** [comment]
how to remove the restriction?

**Comment by oldschool1964** [comment]
I noticed if you decide to turn on Admin Panel > Settings > Web Search > Enable Web Search, then universally, you'll get an error saying: Conversation turn limit exceeded. Max turns: 4. If you turn off your web search, then you'll be able to continue your conversations past 4 interactions.

**Comment by Michaelwucoc** [comment]
Error: Conversation turn limit exceeded. Max turns: 4

Still happen in Knowledge Base Query Mode

Install Pipeline doesn't work for me

**Comment by thiswillbeyourgithub** [comment]
I noticed that the diffferent [filters I found](https://openwebui.com/f/keke/context_length_filter) and [made](https://github.com/thiswillbeyourgithub/openwebui_custom_pipes_filters/blob/main/filters/infinite_chat.py) to automatically modify the body to keep only the last N messages are now broken.

If I modify the body to remove some messages they are back in their entirety in the next filter and ultimately to the LLM.

That looks like a regression no?

**Comment by jstitlow** [comment]
On my setup, the turn limit seems to have been corrupted by the 'Data Vizualization' Function.
 
 Was able to adjust the turn limit in the Chat Control Settings (see attached). 

![image](https://github.com/user-attachments/assets/d8c894f2-5aeb-4943-b751-231aacee72e0)

**Reply by gnodabb** [reply]
I had the "Example Filter" enabled, which automatically limits conversations to 4 or 8 turns. 
Admin Settings -> Functions -> Example Filter: toggle off
This action fixed the issue for me so far.
